content/zh-cn/blog/_posts/2022-08-23-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,402 @@
+-->
+### Ephemeral Containers 成长为 Stable
+
+[Ephemeral Containers](/docs/concepts/workloads/pods/ephemeral-containers/) 是在现有的 pod 中存在有限时间的容器。
+当你需要检查另一个容器，但因为该容器已经崩溃或其镜像缺乏调试工具不能使用 "kubectl exec" 时，它对故障排除特别有用。
+Ephemeral containers 在 Kubernetes v1.23 中成长为 Beta，并在这个版本中，该功能成长为 Stable。
+
+
+<!--
+### Support for cgroups v2 Graduates to Stable
+
+It has been more than two years since the Linux kernel cgroups v2 API was declared stable. With some distributions now defaulting to this API, Kubernetes must support it to continue operating on those distributions. cgroups v2 offers several improvements over cgroups v1, for more information see the [cgroups v2](https://kubernetes.io/docs/concepts/architecture/cgroups/) documentation. While cgroups v1 will continue to be supported, this enhancement puts us in a position to be ready for its eventual deprecation and replacement.
+-->
+### 对 cgroups v2 的稳定支持
+
+自 Linux 内核 cgroups v2 API 宣布稳定以来，已经有两年多的时间了。随着一些发行版现在默认使用该 API，Kubernetes 必须支持它以继续在这些发行版上运行。
+cgroups v2 比 cgroups v1 提供了一些改进，更多信息见 [cgroups v2](https://kubernetes.io/docs/concepts/architecture/cgroups/) 文档。
+虽然 cgroups v1 将继续受到支持，但这一改进使我们能够为其最终的废弃和替代做好准备。
+
+
+<!--
+### Improved Windows support
+
+- [Performance dashboards](http://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019) added support for Windows
+- [Unit tests](https://github.com/kubernetes/kubernetes/issues/51540) added support for Windows
+- [Conformance tests](https://github.com/kubernetes/kubernetes/pull/108592) added support for Windows
+- New GitHub repository created for [Windows Operational Readiness](https://github.com/kubernetes-sigs/windows-operational-readiness)
+-->
+### 改善对 Windows 系统的支持
+- [性能仪表板](http://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019) 增加了对 Windows 系统的支持
+- [单元测试](https://github.com/kubernetes/kubernetes/issues/51540) 增加了对 Windows 系统的支持
+- [一致性测试](https://github.com/kubernetes/kubernetes/pull/108592) 增加了对 Windows 系统的支持
+- 为 [Windows Operational Readiness](https://github.com/kubernetes-sigs/windows-operational-readiness) 创建了新的 GitHub 仓库
+
+<!--
+### Moved container registry service from k8s.gcr.io to registry.k8s.io
+
+[Moving container registry from k8s.gcr.io to registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) got merged. For more details, see the [wiki page](https://github.com/kubernetes/k8s.io/wiki/New-Registry-url-for-Kubernetes-\(registry.k8s.io\)), [annoucement](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ) was sent to the kubernetes development mailing list.
+-->
+### 将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io
+[将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) 的 pr 已经被合并。
```

Comment:
```suggestion
[将容器注册服务从 k8s.gcr.io 迁移至 registry.k8s.io](https://github.com/kubernetes/kubernetes/pull/109938) 的 PR 已经被合并。
```

Reply:
Thank you for your guidance, I have revised it.